.github/workflows/ci.yml

```diff
@@ -45,17 +45,15 @@ jobs:
   try-scenarios:
     name: ${{ matrix.try-scenario }}
     runs-on: ubuntu-latest
-    needs: "test"
+    needs: 'test'
     timeout-minutes: 10
 
     strategy:
       fail-fast: false
       matrix:
         try-scenario:
-          - ember-lts-3.28
```

Comment:
maybe we can keep 3.28+ versions in CI and then it's not breaking change?

Reply:
I was planning to bump ember-resolver to latest, which does break below 4.12

Reply:
That said, if there's a strong reason we need to land some of these changes as a non-major release, we could break it into multiple PRs and do a minor release first.

Reply:
@elwayman02 there is no argument why we should not be doing major.

however, at the same time there are technically no breaking changes, e.g. `ember-resolver` is just a dev dependency and in other addonsI I was downgrading it in ember-try to v9 for <= v4.8 scenarios which allows green CI without any major bumps.

just sharing experience, if you interested you may peek into https://github.com/ember-modifier/ember-modifier/blob/main/test-app/config/ember-try.js#L51

Reply:
That's a fair point. I went ahead and added a resolution on v11 for those scenarios and re-added them.